### ISSUE TITLE
ZeroDivisionError when processing SVS files using deepzoom_tile.py

### ISSUE THREAD

**Issue by mdbloice** [openslide/openslide-python#85]
## Context

**Issue type**: Bug 
**Operating system**: CentOS Linux release 7.6.1810 (Core)
**Platform**: 64-bit x86
**OpenSlide Python version** (`openslide.__version__`): 1.1.1 
**OpenSlide version** (`openslide.__library_version__`): 3.4.1
**Slide format**: SVS

## Details

Hi, 

I am getting a `ZeroDivisionError` when I try to tile an SVS image using `deepzoom_tile.py`: 

```
Traceback (most recent call last):
  File "/usr/lib64/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "deepzoom_tile.py", line 66, in run
    tile = dz.get_tile(level, address)
  File "/usr/local/lib64/python3.6/site-packages/openslide/deepzoom.py", line 158, in get_tile
    tile.thumbnail(z_size, Image.ANTIALIAS)
  File "/usr/local/lib64/python3.6/site-packages/PIL/Image.py", line 2279, in thumbnail
    y = round_aspect(x / aspect, key=lambda n: abs(aspect - x / n))
  File "/usr/local/lib64/python3.6/site-packages/PIL/Image.py", line 2272, in round_aspect
    return max(min(math.floor(number), math.ceil(number), key=key), 1)
  File "/usr/local/lib64/python3.6/site-packages/PIL/Image.py", line 2279, in <lambda>
    y = round_aspect(x / aspect, key=lambda n: abs(aspect - x / n))
ZeroDivisionError: division by zero
```

I was hoping someone could help me out. 

I have tried changing the `tile_size` and `overlap` values but no luck.

Some SVS files that I have work fine, others produce this error.

Thanks for any help.

**Comment by jaharkes** [comment]
This seems relevant, looks like a fix will be part of Pillow 7.2.0.

https://github.com/python-pillow/Pillow/issues/4624

**Comment by mdbloice** [comment]
Thanks a lot, that's likely it. On another machine `deepzoom_tile.py` works as expected so for now I'll downgrade Pillow and update once version 7.2.0 is out. 

Thanks again.
